### DISCUSSION TITLE
Optionally mark the generated bundle as "requiring administrative privileges"

### DISCUSSION THREAD

**Discussion by dan-vasilov** [wixtoolset/issues#8826] [Questions]
Right now the generated bundle manifest only specifies  requestedExecutionLevel  level="asInvoker". Would it be possible to specify during build that the final bundle manifest should contain requestedExecutionLevel  level="requireAdministrator"?

**Comment by robmen** [comment]
Why do you want to do that?

**Comment by dan-vasilov** [comment]
We need this for historical reasons: Windows Explorer automatically adds a shield to the setup icon when the UAC manifest is configured this way, and the people in charge here would like to preserve this behaviour. 
We've been implementing this using a workaround: "use MT.exe to extract existing manifest, use reshacker.exe to put back the edited manifest", and it worked fine for the setup authored using Wix 3. However, we've migrated the setup to Wix 5 and the old workaround no longer works. We've tried again using MT.exe, but MT.exe refuses to merge conflicting manifests or truncates the final .exe file. ResHacker corrupts somehow the bundle. We've tried to write ourselves a small C++ utility, that calls the UpdateResource Win32 API to update the manifest, but the resulted .EXE file was also truncated, just like it happened with mt.exe.
We haven't tried using a PE Editor yet, that will probably be a future step.
However, the logical step here would be to put the desired manifest in the final bundle when the bundle is generated.
Thank you for taking the time to answer, Rob!
